### PR TITLE
add missing parts of the android behavior

### PR DESCRIPTION
**Pull request by boraini**

### CODE REVIEW

**Comment by kuroppoi** [comment]
I think you forgot to push the `fake.json` file

File: gameserver/src/main/java/brainwine/gameserver/entity/npc/job/Job.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import brainwine.gameserver.player.Player;
+import brainwine.gameserver.util.MapHelper;
+
+public abstract class Job {
+    private static Map<String, Job> jobMap = MapHelper.map(
+        String.class, Job.class,
+        "giver", new Giver(),
+        "joker", new Joker(),
+        "crafter", new Crafter()
+    );
```
Comment (kuroppoi) [inline gameserver/src/main/java/brainwine/gameserver/entity/npc/job/Job.java]: Wouldn't it be much more appropriate for `Job` to be an enum in this case?

**Comment by kuroppoi** [comment]
Sorry, haven't really looked into it yet.
Can #58 be closed?

File: gameserver/src/main/java/brainwine/gameserver/entity/npc/job/Job.java (hunk 1 above)
Comment (boraini) [reply gameserver/src/main/java/brainwine/gameserver/entity/npc/job/Job.java]: It is more easily serialized this way.

Comment (kuroppoi) [reply gameserver/src/main/java/brainwine/gameserver/entity/npc/job/Job.java]: I don't know about that, take a look at [Biome.java](https://github.com/kuroppoi/brainwine/blob/master/gameserver/src/main/java/brainwine/gameserver/zone/Biome.java)

**Comment by boraini** [comment]
Just like #61, closing.

**Pull request closed without merging**